httpd24u.spec

```diff
@@ -824,6 +824,12 @@ exit $rv
 
 
 %changelog
+* Fri Oct 05 2021 Reporter4u <reporter4u@gnail.com> - 2.4.50-1
+- Latest upstream
+
+* Fri Oct 05 2021 Reporter4u <reporter4u@gnail.com> - 2.4.49-1
+- Latest upstream
```

Comment:
It's allowed to update directly from 2.4.48 to 2.4.50, you don't have to add a changelog entry for the version we're skipping.

Reply:
I removed 2.4.49 changelog.